Getting_and_Cleaning_Data/Dates_and_Times_with_lubridate/lesson.yaml

```diff
@@ -16,7 +16,7 @@
   Hint: Type Sys.getlocale("LC_TIME") to view your time locale.
 
 - Class: text
-  Output: If the output above is not "en_US.UTF-8", this lesson is not guaranteed to work correctly. Of course, you are welcome to try it anyways. We apologize for this inconvenience.
```

Comment:
The phrase "anyways" is [not common](https://books.google.com/ngrams/graph?content=anyway%2Canyways&year_start=1860&year_end=2000&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Canyway%3B%2Cc0%3B.t1%3B%2Canyways%3B%2Cc0) and it doesn't appear to clarify what's being said.